notebooks/02-ingestion/14-native-filters.ipynb

```diff
@@ -0,0 +1,815 @@
+    "\n",
+    "- Set up a streaming ingestion from Apache Kafka.\n",
+    "- Create two alternative tables from the same topic that contain filtered versions of the source data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bbdbf6ad-ca7b-40f5-8ca3-1070f4a3ee42",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "This tutorial works with Druid 29.0.0 or later.\n",
+    "\n",
+    "Launch this tutorial and all prerequisites using the `all-services` profile of the Docker Compose file for Jupyter-based Druid tutorials. For more information, see the Learn Druid repository [readme](https://github.com/implydata/learn-druid).\n",
+    "   "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5007a243-b81a-4601-8f57-5b14940abbff",
+   "metadata": {},
+   "source": [
+    "## Initialization\n",
+    "\n",
+    "The following cells set up the notebook and learning environment ready for use."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4439783f-16cc-4f67-bd50-bdfd4539e9b0",
+   "metadata": {},
+   "source": [
+    "### Set up a connection to Apache Druid\n",
+    "\n",
+    "Run the next cell to set up the Druid Python client's connection to Apache Druid.\n",
+    "\n",
+    "If successful, the Druid version number will be shown in the output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1ec783b-df3f-4168-9be2-cdc6ad3e33c2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import druidapi\n",
+    "import os\n",
+    "\n",
+    "druid_headers = {'Content-Type': 'application/json'}\n",
+    "\n",
+    "if 'DRUID_HOST' not in os.environ.keys():\n",
+    "    druid_host=f\"http://localhost:8888\"\n",
+    "else:\n",
+    "    druid_host=f\"http://{os.environ['DRUID_HOST']}:8888\"\n",
+    "\n",
+    "print(f\"Opening a connection to {druid_host}.\")\n",
+    "druid = druidapi.jupyter_client(druid_host)\n",
+    "display = druid.display\n",
+    "sql_client = druid.sql\n",
+    "status_client = druid.status\n",
+    "\n",
+    "status_client.version"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "872dd1b5-419d-4201-8061-5339c8462386",
+   "metadata": {},
+   "source": [
+    "### Set up a connection to Apache Kafka\n",
+    "\n",
+    "Run the next cell to set up the connection to Apache Kafka."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "358b18a2-1aea-4538-a711-476877417509",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if 'KAFKA_HOST' not in os.environ.keys():\n",
+    "   kafka_host=f\"http://localhost:9092\"\n",
+    "else:\n",
+    "    kafka_host=f\"{os.environ['KAFKA_HOST']}:9092\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2efdbee0-62da-4fd3-84e1-f66b8c0150b3",
+   "metadata": {},
+   "source": [
+    "### Set up a connection to the Data Generator\n",
+    "\n",
+    "Run the next cell to set up the connection to the Data Generator."
```

Comment:
Is data generator a proper now? If so, it should be capitalized everywhere else. Otherwise, use lowercase.

Reply:
Switched to lower case throughout.